README.md

```diff
@@ -71,6 +71,7 @@ an [issue](https://github.com/ionic-team/ionic/issues/new) on this repository.
 
 Already have an Ionic app? These guides will help you migrate to the latest versions.
 
+* [Migrate from v6 to v7](https://ionicframework.com/docs/v7/updating/7-0)
 * [Migrate from v5 to v6](https://ionicframework.com/docs/reference/migration#migrating-from-ionic-5x-to-ionic-6x)
```

Comment:
Shouldn't these URLs be updated too? https://ionicframework.com/docs/v7/updating/6-0

Reply:
Those URLs will be 301 redirected when v7 comes out so the URLs still work, but probably a good idea to update them anyways (https://github.com/ionic-team/ionic-nginx/pull/22)